Composer/packages/client/src/constants.tsx

```diff
@@ -200,9 +200,9 @@ export const BotStatusesCopy = {
 export const DialogCreationCopy = {
   get CREATE_OPTIONS() {
     return {
-      title: formatMessage('Your new Azure Bot is available in Composer'),
+      title: formatMessage('Choose a bot project for your Azure Bot'),
       subText: formatMessage(
-        'The Azure Bot created in Azure Bot Services contains bot resources that can be used as the basis for a new bot, or to add or replace resources of an existing bot.'
+        'Use your Azure Bot resource with a new or existing bot project. You can publish your bot project directly to Azure, using the publishing profile created for you.'
```

Comment:
Unnecessary coma. 
```suggestion
        'Use your Azure Bot resource with a new or existing bot project. You can publish your bot project directly to Azure using the publishing profile created for you.'
```

Reply:
Thanks for catching that; I'd blindly copied the text I was handed without proofreading it first.